scripts/microbe_stage/microbe_replacement.lua

```diff
@@ -64,5 +54,20 @@ function MicrobeReplacementSystem:activate()
     end
 end
 
+--Faux-latin name generation routine (Move to own file eventually?)
+function MicrobeReplacementSystem:generateSpeciesName()
+
+    --prefix,cofix,suffix list
+    speciesNamePrefix = {' Ce', ' Ar',' Sp', ' Th',' Co', ' So', ' Pu', ' Cr', ' Cy', ' Gr', ' Re', ' Ty', ' Tr', ' Ac',' Pr' }
+    speciesNameCofix = { 'nan', 'mo', 'na', 'yt', 'yn', 'il', 'li','le', 'op', 'un', 'rive','ec', 'ro','lar','im' }
+    speciesNameSuffix = { 'pien', 'olera', 'rius', 'nien', 'ster', 'ilia', 'canus', 'tus', 'cys','ium','um'}
```

Comment:
These should be kept in constant instead of a local variable. Not necessarily a global constant though.